### PR TITLE
Feature: Parsing optional CONFIG values + log CONFIG at start

### LINKED ISSUE #5: Invision scraper - add more EXPECTED_URL_PARTS
Need more EXPECTED_URL_PARTS - found "/forum" as valid and needed part

### PR DESCRIPTION
**Pull request by Samox1**

### CODE REVIEW

File: invision-forum-crawler.py [hunk 1]
```diff
@@ -80,7 +80,8 @@
 DATASET_NAME = "dataset_name" # for example: forum_website_pl_corpus
 DATASET_DESCRIPTION = f"Collection of forum discussions from {DATASET_URL}" # also change this according to the forum name
 LICENSE = "(c) www.forumaddress.pl" # forum address
-EXPECTED_URL_PARTS = ['/topic','/temat', '/thread'] # we're targeting the full topics to optimize the performance
+
+EXPECTED_URL_PARTS = ['/topic','/temat', '/thread', '/forum'] # we're targeting the full topics to optimize the performance
```
Comment (pawelkiszczak) [inline invision-forum-crawler.py:103]: Is the '/forum' part needed? The script does only go through the topics, not whole forums/subforums.

File: invision-forum-crawler.py [hunk 2]
```diff
@@ -273,6 +274,7 @@ def get_item_text(url: str) -> str:
                         text += comment.text.strip() + "\n"
                     #for i in page_nav_results:
                     #    text += i.text.strip()+"\n"
+                    time.sleep(TIME_SLEEP)
```
Comment (pawelkiszczak) [inline invision-forum-crawler.py:299]: Ok, necessary addition.

**Review by pawelkiszczak — COMMENTED** [review]
@Samox1 Please take a look at the comments I've made.

File: invision-forum-crawler.py [hunk 3]
```diff
@@ -64,27 +65,46 @@
...
-# TODO!: Merged *.jsonl.zst - tweak what should be in final archive
-
 # TODO 1: Create generator to iterate over file (csv) with links --> to minimize memory footprint
 # TODO 1: - for now: pandas.DataFrame with urls (in memory) is duplicated and split for each worker, 
 # TODO 1: --> so we got x2 mem of this dataframe ('cos each subprocess got a chunk) + memory for workers
 
 # TODO 2: Each worker could export dict with: url, flag_visited, flag_skipped - for now its tricky process
 
 
+parser = argparse.ArgumentParser(description='Crawler and scraper - using sitemaps (in XML) and scrap text from Invision forums')
```
Comment (pawelkiszczak) [inline invision-forum-crawler.py:75]: Parser part is necessary, great addition for future features.

File: invision-forum-crawler.py (hunk 1 above)
Comment (Samox1) [reply invision-forum-crawler.py:103]: Found forums where early topics were under '/forum'

**Review by pawelkiszczak — APPROVED** [review]
Changes approved

**Pull request merged**